src/cell_gater/widgets/scatter_widget.py

```diff
@@ -158,46 +175,61 @@ def _set_samples_dropdown(self) -> None:
             if len(self.model.samples) > 0:
                 self.sample_selection_dropdown.addItems([None])
                 self.sample_selection_dropdown.addItems(self.model.samples)
+    
+    def _on_y_axis_changed(self):
+        """Set active y-axis and update the scatter plot."""
+        self.model.active_y_axis = self.choose_y_axis_dropdown.currentText()
+        self.update_plot()
 
+    def update_plot(self):
+        self.scatter_canvas.plot_scatter_plot(self.model)
```

Comment:
```suggestion
        self.scatter_canvas.plot_scatter_plot()
```
Model is passed on between the different widgets and thus should have the same memory address. We can write tests for this later.